ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -725,6 +725,19 @@ tests:
     - ref: idp-htpasswd
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy
+- as: vsphere-ipi-proxy-dualstack-privmaryv6-f28-disasterrecovery
+  cron: 7 1 * * 2,6
```

Comment:
The test name is `f28` which indicate its test frequency is every 28 days, while the cron entry means twice every week.  Perhaps make them in sync ?

Reply:
Updated.

Reply:
I think `vsphere-ipi-proxy-dualstack-privmaryv6` can also cover `vsphere-ipi-proxy` for disasterrecovery. Can we use this one to replace the other one ? CC @jianlinliu

Reply:
And the install workflow is "cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack", where indicated it is a proxy install ?

Reply:
so update "vsphere-ipi-proxy-dualstack-privmaryv6" to vsphere-ipi-dualstack-privmaryv6 could be perfect, exact?

Reply:
I searched the existing ci jobs using "workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack", the ci job name is "vsphere-ipi-ovn-dualstack-privmaryv6-f28", so the answer is "yes".

And I suggest to always get the exact job configuration from the existing ci job for your new job, e.g: I compared the setting in `vsphere-ipi-ovn-dualstack-privmaryv6-f28` with yours, sounds the cluster_profile and leases are wrong in your job configurations.